bounties_api/std_bounties/client_helpers.py

```diff
@@ -47,10 +48,17 @@ def map_bounty_data(data_hash, bounty_id):
         ipfs_hash = 'invalid'
 
     data = json.loads(data_JSON)
-    metadata = data.get('meta', {})
+
+    meta = data.get('meta', {})
+
     if 'payload' in data:
         data = data.get('payload')
 
+    metadata = data.get('metadata', {})
+
+    formattedExperienceLevel = metadata.get('experienceLevel', '').lower().strip().capitalize()
+    metadata.update({ 'experienceLevel': rev_mapped_difficulties.get(formattedExperienceLevel, 0)})
```

Comment:
@c-o-l-o-r instead of 0 - can you use the variable?  This allows us to programmatically update if there's ever an issue.

Reply:
it has been updated, good call!